src/services/review/ReviewRequestService.ts

```diff
@@ -512,7 +512,7 @@ export default class ReviewRequestService {
       // Using map here to allow creations to be done concurrently
       // But we do not actually need the result of the view creation
       requestIdsToMarkAsViewed.map(async (requestId) =>
-        this.reviewRequestView.create({
+        this.reviewRequestView.upsert({
```

Comment:
hmm, maybe the correct thing to do would be to use a transaction here. wdyt?

Reply:
[ok]([5a7a311](https://github.com/isomerpages/isomercms-backend/pull/861/commits/5a7a311d962761db16e86d73811e18fa098f8b06))